src/features/island/farmerQuest/FarmerQuest.tsx

```diff
@@ -4,43 +4,70 @@ import { GRID_WIDTH_PX, PIXEL_SCALE } from "features/game/lib/constants";
 
 import farmerNpc from "assets/npcs/idle.gif";
 import shadow from "assets/npcs/shadow.png";
-import close from "assets/icons/close.png";
 import island from "assets/land/islands/farmer_island.webp";
 import { Modal } from "react-bootstrap";
-import { Panel } from "components/ui/Panel";
-import { Button } from "components/ui/Button";
-import { QuestProgress } from "./components/QuestProgress";
+import { Quest } from "features/game/expansion/components/Quest";
+import appleTree from "assets/fruit/apple/apple_tree.png";
+import orangeTree from "assets/fruit/orange/orange_tree.png";
+import blueberryBush from "assets/fruit/blueberry/blueberry_bush.png";
 
 export const FarmerQuest: React.FC = () => {
-  const [state, setState] = useState<"closed" | "introduction" | "progress">(
-    "closed"
-  );
+  const [showModal, setShowModal] = useState(false);
 
-  const ModalContent = () => {
-    if (state === "introduction") {
-      return (
-        <div className="p-1">
-          <p>Howdy farmer!</p>
-          <p className="mt-4">
-            {`I'm looking for expert farmers to trial some free Bumpkin clothing.`}
-          </p>
-          <p className="mt-4 mb-2">
-            If you prove you are a hardworking Bumpkin, I will give you rare
-            Farmer clothing.
-          </p>
-          <Button disabled onClick={() => setState("progress")}>
-            Continue
-          </Button>
+  const ModalDescription = () => {
+    return (
+      <>
+        <p className="mb-3">
```

Comment:
Can we replace the jsx fragment into div to wrap the long text and add more margin?